### DISCUSSION TITLE
Keyboard input to rotate/zoom 3D model on Android

### DISCUSSION THREAD

**Discussion by kelvinwatson** [google/filament#5259] [Ideas]
Is there any interest in introducing functionality to allow keyboard input (arrows) to rotate the 3d model on Android? I would be happy to contribute to the code base if this is desired. This would be a good consideration for accessibility support.

**Comment by romainguy** [comment]
That'd be really useful as part of our utils library yes.

**Reply by kelvinwatson** [reply]
Great! I'll be submitting a PR for this in the coming weeks.
